pkg/storage/azurecdn/storage.go

```diff
@@ -8,26 +8,26 @@ import (
 	"net/url"
 
 	"github.com/Azure/azure-storage-blob-go/2017-07-29/azblob"
-	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/config/env"
-	multierror "github.com/hashicorp/go-multierror"
+	m "github.com/gomods/athens/pkg/storage/module"
```

Comment:
something more descriptive please

Reply:
module would cause conflicts, but for example modupl. or uploader as we use this only to m.Upload.
moduploader.Upload?
just few ideas, none of those seems 100% fine